specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/applicationSecurityGroup.json

```diff
@@ -174,6 +174,56 @@
         "x-ms-examples": {
           "Create application security group": { "$ref": "./examples/ApplicationSecurityGroupCreate.json" }
         }
+      },
+      "patch": {
+        "tags": [
+          "applicationSecurityGroups"
+        ],
+        "operationId": "ApplicationSecurityGroups_UpdateTags",
+        "description": "Updates an application security group's tags.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "applicationSecurityGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the application security group."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./network.json#/definitions/TagsObject"
+            },
+            "description": "Parameters supplied to update application security group tags."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
nit: please make sure this really is a long running opreration

Reply:
I was using vnet PATCH as an example, this is what it does. We should aim for consistency across our resources, right?